benchmark/stats/util.go

```diff
@@ -16,6 +16,8 @@
  *
  */
 
+// Package stats registers benchmark statistics. utils.go provide methods to
+// add statistics for benchmarks
```

Comment:
Package level comments are generally put into a single file. So, this one isn't necessary.